addons/product_margin/security/ir.model.access.csv

```diff
@@ -1 +1,2 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_report_product_margin,report.product.margin,model_report_product_margin,,1,0,0,0

```

Comment:
@gan-odoo If I'm asking the question, maybe it is due to the face that default access rules are not the smartest idea... think about it...